src/borg/cache.py

```diff
@@ -225,6 +225,9 @@ def recanonicalize_relative_location(cache_location, repository):
 def cache_dir(repository, path=None):
     return path or os.path.join(get_cache_dir(), repository.id_str)
 
+def files_cache_name():
+    return 'files' + os.environ.get('BORG_FILES_CACHE_SUFFIX', '')
```

Comment:
if you set `BORG_FILES_CACHE_SUFFIX=foo`, this will be `filesfoo`.

works, but is a bit ugly. how about adding a dot in between IF the env var is not empty?

Reply:
Why not replacing 'files' completely with a env variable `BORG_FILES_CACHE_NAME=files`?

Reply:
guess that could be harmful, just imagine someone setting it to `chunks` (or some other already taken name, or empty).

besides that, the docs talk about the "files cache", so it should be recognizable as that.

Reply:
Personally, I am using a hyphen as my lead character for `BORG_FILES_CACHE_SUFFIX`. A dot historically implies a file type. Does a hyphen seem acceptable to you?

Reply:
### examples from borg cache:

`chunks` vs. `chunks.archive.d` - uses a `.` to separate different parts of file name.

### examples from borg repo:

`hints.NN`, `index.NN`, `integrity.NN` (with NN being a number) - also uses a dot to separate parts of filename. `NN` is not the file type here, but the segments transaction id.

### examples from borg config:

`key-type`, `manifest-timestamp` - a dash `-` is just used for readability, not as a separator to append a scope.

so guess using a dot is more consistent with what we already have.